test/SourceKit/CursorInfo/cursor_vardecl_across_fallthrough.swift

```diff
@@ -0,0 +1,57 @@
+
+enum X {
+  case first(Int, String)
+  case second(Int, String)
+  case third(Int, String)
+  case fourth(Int, String)
+  case fifth(Int, String)
+}
+
+let p = X.first(3, "hello")
+
+switch p {
+  case .first(let x, let y)
+    print("foo \(x) \(y)")
+    fallthrough
+  case .second(let x, let y), .third(let x, let y):
+    print("bar \(x) \(y)")
+  default:
+    print("other")
+}
+
+// RUN: %sourcekitd-test -req=cursor -pos=13:19 %s -- %mcp_opt %s | %FileCheck -check-prefixes=CHECKX,CHECK1DECL %s
+// RUN: %sourcekitd-test -req=cursor -pos=14:18 %s -- %mcp_opt %s | %FileCheck -check-prefixes=CHECKX,CHECK1REF %s
+
+// CHECK1DECL: source.lang.swift.decl.var.local (13:19-13:20)
+// CHECK1REF: source.lang.swift.ref.var.local (13:19-13:20)
+
+// RUN: %sourcekitd-test -req=cursor -pos=16:20 %s -- %mcp_opt %s | %FileCheck -check-prefixes=CHECKX,CHECK2DECL %s
+// RUN: %sourcekitd-test -req=cursor -pos=16:42 %s -- %mcp_opt %s | %FileCheck -check-prefixes=CHECKX,CHECK2DECL2 %s
+// RUN: %sourcekitd-test -req=cursor -pos=17:18 %s -- %mcp_opt %s | %FileCheck -check-prefixes=CHECKX,CHECK2REF %s
+
+// CHECK2DECL: source.lang.swift.decl.var.local (16:20-16:21)
+// CHECK2DECL2: source.lang.swift.decl.var.local (16:42-16:43)
+// CHECK2REF: source.lang.swift.ref.var.local (16:20-16:21)
```

Comment:
@gottesmm This is the location jump-to-definition will land on when invoked on the reference on line 17. Once you have your changes in I expect the `(16:20-16:21)` on this line to change to `(16:42-16:43)`. I.e. cursor info should say the second 'let x' on line 16 is the one being referred to.